app/src/main/java/com/example/steamapp/FragmentDetails.kt

```diff
@@ -0,0 +1,78 @@
+package com.example.steamapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import coil.load
+import com.example.steamapp.databinding.FragmentDetailsBinding
+
+class FragmentDetails : Fragment() {
```

Comment:
по кодстайлу, сначала имя потом Fragment 
```suggestion
class DetailsFragment : Fragment() {
```